### DISCUSSION TITLE
Some script run too early that "document" is not exist yet

### DISCUSSION THREAD

**Discussion by 0rt** [xiaoxiaoflood/firefox-scripts#147] [General Support]
Mostly those with "@onlyonce" tag, I don't know why, but the error message suggest it's run too early, like "document is null" "alert is not function"
If I remove the "@onlyonce" line, it seem work as supposed.
Because the document is not exist, I can't add a listner for document loaded. 

FX 101, your userchrome.js loader

**Comment by xiaoxiaoflood** [comment]
You probably should try with `window.` prefix, like `window.document` and `window.alert`.

Also `window.addEventListener('DOMContentLoaded', () => {/*function code*/})` (or `load` instead of `DOMContentLoaded`).
